CONTRIBUTORS.txt

```diff
@@ -14,6 +14,16 @@ Text conversion:
   ShadowOne333
   PhoenixBound
   Pidgin
+  Sterophonick
+  Karmageddon
 
 Special thanks:
-  TragicManner
\ No newline at end of file
+  TragicManner
+  Nintendo

```

Comment:
I'd lean towards not including the companies in the contributors

Reply:
Ditto.

Reply:
Understandable.